pkg/targetallocator/adapters/config_to_promConfig.go

```diff
@@ -0,0 +1,62 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
I'm kinda bothered by the name of this file, and would prefer `_config.go` instead of `Config.go`, but not sure about the Go conventions for this case.

Reply:
I think your suggestion of using `_config.go` sits better with the Go conventions than the name `Config.go`. Changed it to `_config.go`.